MobileBuy/buy3/src/main/java/com/shopify/buy3/GraphClient.kt

```diff
@@ -178,12 +192,12 @@ class GraphClient private constructor(
          *
          * @return configured [GraphClient]
          */
-        fun build(): GraphClient {
+        fun build(locale: String?): GraphClient {
             val httpCache = httpCacheConfig.let { config ->
                 when (config) {
                     is HttpCacheConfig.DiskLru -> {
                         val version = BuildConfig.VERSION_NAME
-                        val tmp = (endpointUrl.toString() + "/" + version + "/" + accessToken).toByteArray(Charset.forName("UTF-8"))
+                        val tmp = (endpointUrl.toString() + "/" + version + "/" + accessToken + "/" + locale).toByteArray(Charset.forName("UTF-8"))
```

Comment:
@sav007 is this the only change needed to make caching work with different locales?